packages/smooth_app/lib/main.dart

```diff
@@ -134,111 +131,54 @@ class SmoothAppGetLanguage extends StatelessWidget {
   }
 }
 
-class SmoothApp extends StatelessWidget {
-  static const double _navigationIconSize = 32.0;
-  static const double _navigationIconPadding = 5.0;
+class SmoothApp extends StatefulWidget {
+  @override
+  _SmoothAppState createState() => _SmoothAppState();
+}
 
+/// This is the private State class that goes with MyStatefulWidget.
+class _SmoothAppState extends State<SmoothApp> {
   @override
   Widget build(BuildContext context) {
     final UserPreferences userPreferences = context.watch<UserPreferences>();
     final bool mlKitState = userPreferences.getMlKitState();
-    return SmoothNavigationLayout(
-      layout: SmoothNavigationLayoutModel(
-        screens: <SmoothNavigationScreenModel>[
-          _generateScreenModel(
-            context,
-            mlKitState,
-            'Choose',
-            ChoosePage(),
-            svg: 'assets/ikonate_thin/search.svg',
-          ),
-          /*
-          _generateScreenModel(
-            context,
-            mlKitState,
-            'assets/ikonate_thin/organize.svg',
-            'Organize',
-            OrganizationPage(),
+    return Scaffold(
+      body: Center(
+        child: Container(),
```

Comment:
That doesn't make sense to have this `Container`. But this was the tricky part: from a fake bottom navigation bar to a real app bar.